tsconfig.json

```diff
@@ -25,6 +25,6 @@
       "@/utils/*": ["src/utils/*"]
     }
   },
-  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "**/*.ts", "**/*.tsx", "src/components/Search/Search.test.tsx", "src/components/Search/Search.tsx"],
```

Comment:
These 2 patterns seem like they are already part of the `"**/*.tsx"` pattern.

Reply:
@Duslerke This should be fixed now!